packages/auth/src/wallet.ts

```diff
@@ -50,27 +36,17 @@ export let walletSelectorComponents: WalletSelectorComponents  = {
 */
 export const setupWalletSelectorComponents = async (): Promise<WalletSelectorComponents> => {
   const selector = await setupWalletSelector({
-    network: NEAR_NETWORK,
-    debug: NEAR_WALLET_SELECTOR_DEBUG,
+    network: mbjs.keys.network,
+    debug: mbjs.keys.debugMode,
     modules: [
       ...(await setupDefaultWallets()),
-      setupNearWallet(),
-      setupMeteorWallet(),
-      setupSender(),
-      // setupHereWallet(),
-      // setupMathWallet(),
-      // setupNightly(),
-      // setupWelldoneWallet(),
-      // setupLedger(),
+      ...SUPPORTED_NEAR_WALLETS,
     ],
   });
 
-  const defaultMintbaseContract = NEAR_NETWORK === 'testnet'
-    ? DEFAULT_MINTBASE_CONTRACT_TESTNET
-    : DEFAULT_MINTBASE_CONTRACT_MAINNET;
 
   const modal = setupModal(selector, {
-    contractId: NEAR_LOGIN_CONTRACT_ID || defaultMintbaseContract,
+    contractId: mbjs.keys.contractAddress ?? mbjs.keys.mbContract,
```

Comment:
To clarify your question asked in dev channel regarding this option, when you login with a contract you are giving an allowance of 0.25 N that can be used for calls on that contract without signing.  We enforce signing for nearly all transactions anyway, so it's not the most relevant to set - but the user does see that when logging in so it's nice to have it configurable.

Reply:
followed the previous logic